docs/pages.jl

```diff
@@ -10,8 +10,8 @@ pages = Any["Home" => "index.md",
             "Catalyst Applications" => Any["catalyst_applications/simulation_structure_interfacing.md",
                                            "catalyst_applications/advanced_simulations.md",
                                            "catalyst_applications/homotopy_continuation.md",
-                                           "catalyst_applications/bifurcation_diagrams.md",
-                                           "catalyst_applications/parameter_estimation.md"],
+                                           "catalyst_applications/bifurcation_diagrams.md"],
+            "Inverse Problems" => Any["inverse_problems/parameter_estimation.md"],
             "Example Networks" => Any["example_networks/basic_CRN_examples.md",
                                       "example_networks/hodgkin_huxley_equation.md",
                                       "example_networks/smoluchowski_coagulation_equation.md"],
```

Comment:
```suggestion
            "Example Networks" => Any["example_networks/basic_CRN_examples.md",
                                                             "example_networks/hodgkin_huxley_equation.md",
                                                             "example_networks/smoluchowski_coagulation_equation.md"],
             "Inverse Problems" => Any["inverse_problems/parameter_estimation.md"],
```
I think it makes more sense to show examples of building models before showing an analysis that uses models.

Reply:
Otherwise looks good.

Reply:
I don't think any of the models under "Inverse Problems" would depend on those in "Example Networks". My thought was that the "Inverse Problem" section is basically equivalent to "Catalyst Application" section, i.e. showing how to do certain analysis (but cut off into its own section to avoid creating a really long one).